manage_arkime/core/price_report.py

```diff
@@ -0,0 +1,79 @@
+from core.capacity_planning import ClusterPlan
+from core.user_config import UserConfig
+from typing import Dict
+from dataclasses import dataclass
+import math
+
+AWS_HOURS_PER_MONTH=730
+AWS_SECS_PER_MONTH=60*60*AWS_HOURS_PER_MONTH
+
+us_east_1_prices: Dict[str, float] = {
+    "t3.small.search": 0.0360 * AWS_HOURS_PER_MONTH,
+    "m5.large.search": 0.1420 * AWS_HOURS_PER_MONTH,
+    "m6g.large.search": 0.1280  * AWS_HOURS_PER_MONTH,
+    "c6g.2xlarge.search": 0.4520 * AWS_HOURS_PER_MONTH,
+    "r6g.large.search": 0.1670 * AWS_HOURS_PER_MONTH,
+    "r6g.2xlarge.search": 0.6690 * AWS_HOURS_PER_MONTH,
+    "r6g.4xlarge.search": 1.3390 * AWS_HOURS_PER_MONTH,
+    "r6g.12xlarge.search": 4.0160 * AWS_HOURS_PER_MONTH,
+    "m5.xlarge": 0.1920 * AWS_HOURS_PER_MONTH,
+
+    "s3-STANDARD-50-GB": 0.023, # https://aws.amazon.com/s3/pricing/
+    "s3-STANDARD-450-GB": 0.022,
+    "s3-STANDARD-REST-GB": 0.021,
+    "ebs-GB": 0.10, # https://aws.amazon.com/ebs/pricing/
+    "gwlb-GB": 0.004, # https://aws.amazon.com/elasticloadbalancing/pricing/?nc=sn&loc=3
+    "gwlbe-GB": 0.0035, # https://aws.amazon.com/privatelink/pricing/
+
+    "fargate": 0.04048 * AWS_HOURS_PER_MONTH, # https://aws.amazon.com/fargate/pricing/
+
+    "trafficmirror": 0.015 * AWS_HOURS_PER_MONTH, # https://aws.amazon.com/vpc/pricing/
+}
+
+
+@dataclass
+class PriceReport:
+    plan: ClusterPlan
+    config: UserConfig
+    prices = us_east_1_prices # ALW - Not sure how to do type hint, was getting errors
+
+    total: float = 0 # ALW - Not sure how to do a class variable that isn't passed in
```

Comment:
Seeing this, I think we might be better off ditching the @dataclass decorator and making a normal Python class like so:

```
class PriceReport
    def __init__(self, plan: ClusterPlan, config: UserConfig, prices: Dict[str, float] = None):
        self._plan = plan
        self._config = config
        self._prices = _prices if _prices else US_EAST_1_PRICES

        self._total = 0

    .
    .
    .
```